layers/+lang/typescript/packages.el

```diff
@@ -2,21 +2,42 @@
 ;;
 ;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
 ;;
-;; Author: Chris Bowdon <c.bowdon@bath.edu>
-;; URL: https://github.com/syl20bnr/spacemacs
+;; Author: JAremko <w3techplaygound@gmail.com> 
+;; URL: https://github.com/JAremko/spacemacs-pr
 ;;
```

Comment:
@robbyoconnor  Those was `packages.el` credentials - it was completely scrapped.  And if something breaks it will be better if users know to whom send email.

 But sure.  I don't mind  :+1: